spaces/xfinance/index.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "xfinance",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "XFI",
+  "defaultView": "all",
+  "token": "0x5BEfBB272290dD5b8521D4a938f6c4757742c430",
+  "core": [
+    "0xF104f1d4F71920334DC765aD2194B8F450567DF2"
+  ],
+  "min": 20,
+  "invalid": [],
+  "skin": "xfi",
```

I you want to use the `aave` skin, add it on your skin field.
```suggestion
  "skin": "aave",
```